doc/releases/changelog-dev.md

```diff
@@ -136,6 +136,25 @@
 
 <h3>Deprecations</h3>
 
+* In-place inversion is now deprecated. This includes `op.inv()` and `op.inverse=value`. Please
+  use `qml.adjoint` instead. Support for these methods will remain till v0.28.
```

Comment:
For non-unitaries (e.g., channels) there is a difference between the inverse and adjoint. What could users use going forward there to get the inverse for such operators? Worth mentioning.

If we've found that getting the inverse of non-unitaries is not a use case worth supporting natively in PennyLane, then it is also worth mentioning.

Edit: going fwd I'd guess it's ``op ** -1``.